### PR TITLE
merge tslint-config from master

### PR DESCRIPTION
**Pull request by shoota**
Connect to #1

### CODE REVIEW

File: packages/tslint-config/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+npm test
+```
+
+## License
+
+MIT
+
+[david-dm-image]: https://david-dm.org/uncovertruth/tslint-config.svg
+[david-dm-url]: https://david-dm.org/uncovertruth/tslint-config
+[dev-david-dm-image]: https://david-dm.org/uncovertruth/tslint-config/dev-status.svg
```
Comment (9renpoto) [inline packages/tslint-config/README.md]: not found david

File: packages/tslint-config/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@uncovertruth/tslint-config",
+  "description": "UNCOVERTRUTH TypeScript Linter rule",
+  "version": "1.3.0",
+  "author": "UNCOVER TRUTH Inc",
+  "bugs": {
+    "url": "https://github.com/uncovertruth/tslint-config/pulls"
```
Comment (9renpoto) [inline packages/tslint-config/package.json]: fix repo url

File: packages/tslint-config/README.md (hunk 1 above)
Comment (shoota) [reply packages/tslint-config/README.md]: david is alive, and delete greenkeeper.

Comment (shoota) [reply packages/tslint-config/README.md]: in fact, david is dead....

**Pull request merged**